actions/tor

```diff
@@ -68,6 +69,9 @@ def parse_arguments():
 
 def subcommand_setup(_):
```

Comment:
We should also stop and disable the normal "tor" service, at the beginning of setup.